src/main.zig

```diff
@@ -7157,6 +7157,35 @@ fn accessLibPath(
         return true;
     }
 
+    if (target.os.tag == .openbsd and link_mode == .Dynamic) {
+        test_path.clearRetainingCapacity();
+        var dir = fs.cwd().openDir(lib_dir_path, .{}) catch {
+            return false;
+        };
+        defer dir.close();
+        var iter = dir.iterate();
+
+        const allocator = @import("std").heap.page_allocator;
```

Comment:
This should use the same allocator that the array lists are using.

Reply:
i patched by using test_path.allocator